code/modules/multiz/structures.dm

```diff
@@ -108,6 +107,34 @@
 /obj/structure/multiz/ladder/attack_generic(var/mob/M)
 	attack_hand(M)
 
+/obj/structure/multiz/ladder/proc/throw_through(var/obj/item/C, var/mob/throw_man)
+	if(istype(throw_man,/mob/living/carbon/human))
+		var/mob/living/carbon/human/user = throw_man
+		var/through =  istop ? "down" : "up"
+		user.visible_message("<span class='warning'>[user] takes position to throw [C] [through] \the [src].</span>",
+		"<span class='warning'>You take position to throw [C] [through] \the [src].</span>")
+		if(do_after(user, 10))
+			user.visible_message("<span class='warning'>[user] throws [C] [through] \the [src]!</span>",
+			"<span class='warning'>You throw [C] [through] \the [src]</span>")
```

Comment:
Span macros would be better for this

Reply:
I also care about this, please use span macros.

Reply:
what are spam macros

Reply:
I understand they condense, but uh

Reply:
Sort of SPAN_WARNING("text here")

Reply:
done